virtme/commands/run.py

```diff
@@ -968,6 +968,28 @@ def cleanup_console_script():
     qemuargs.extend(["-device", "vhost-vsock-pci,guest-cid=%d" % args.port])
 
 
+def ssh_client(args):
+    remote_cmd = args.remote_cmd if args.remote_cmd else ""
+    cmd = "ssh -p %d localhost %s" % (args.port, remote_cmd)
+    if args.dry_run:
+        print(cmd)
+    else:
+        os.system(cmd)
```

Comment:
(out of curiosity, why `os.system()`, and not `os.execvp()`? Should we change what is done in `console_client()`)

Reply:
`os.system()` was just easier, I was planning to change it to `os.execvp()` but I forgot, I'll fix it. :)